_release/bat/README.md

```diff
@@ -20,10 +20,10 @@ existing test case runner, test-cases.
 
 ## Set up
 
-The BAT tests require a running ciao cluster to execute.  This can be a
+The BAT tests should run from the controller node of a ciao cluster.  This can be a
```

Comment:
As you say in the commit message the real issue is not that the BAT tests need to be run from the controller node.  Rather it's that the controller node has all the pieces in place for the tests to run successfully.

I'd leave the existing text the way it is an add an extra paragraph after the Single VM URL.

> The BAT tests have some dependencies.  The device on which they are run must have qemu-img installed and must also have access to the ceph cluster.  The controller node in a ciao cluster fulfills both of these requirements so it is often easiest to run the BAT tests from this node.  There is only one node in Single VM clusters, so when using Single VM, simply run the BAT tests from the device on which you ran setup.sh.